dev/test/transaction.ts

```diff
@@ -106,7 +106,7 @@ function commit(
 
 function rollback(
   transaction?: Uint8Array | string,
-  err?: Error
+  error?: Error
```

Comment:
optional: change naming for `commit()` as well so they're all in sync ([link](https://github.com/googleapis/nodejs-firestore/blob/8f41b37fd766b18113401c7324e319b52b38cc96/dev/test/transaction.ts#L73))

Reply:
Done. Good catch.